MathLib/CMakeLists.txt

```diff
@@ -60,25 +63,34 @@ add_library(MathLib STATIC ${SOURCES})
 
 set_target_properties(MathLib PROPERTIES LINKER_LANGUAGE CXX)
 
-target_link_libraries(MathLib INTERFACE
-	GeoLib
-	logog
-)
+if (CVODE_FOUND)
+	target_link_libraries(MathLib INTERFACE
+		GeoLib
+		logog
+		sundials_cvode
+		sundials_nvecserial
+	)
+else()
```

Comment:
- Skip the else branch and add sundials libraries as for example lapack libraries after the default target_link_libs(GeoLib logog)
 - If the sundials libraries' names are available in a variable like SUNDIALS_LIBRARIES, it would be nicer to use that variable instead of naming them explicitly